app/controllers/api/verify/document_capture_controller.rb

```diff
@@ -0,0 +1,79 @@
+module Api
+  module Verify
+    class DocumentCaptureController < BaseController
+      self.required_step = 'document_capture'
+      include ApplicationHelper
+
+      def create
+        result = @form ||= Idv::ApiDocumentVerificationForm.new(
+          verify_params,
+          liveness_checking_enabled: liveness_checking_enabled?,
+          analytics: analytics,
+        ).submit
+
+        if result.success?
+          analytics.idv_address_submitted(**result.to_h)
```

Comment:
Thanks for adding analytics! Some notes:
1. we should log in both success and error cases, so we should move this above the `if success?` check
2. `idv_address_submitted` is probably the wrong event, since this is document capture. We might have to create a new analytics event type for this if there isn't a good match already